tests/dualtor_mgmt/test_egress_drop_nvidia.py

```diff
@@ -0,0 +1,247 @@
+            lower_tor_vlan_ptf_ports.append(lower_tor_mg_facts['minigraph_ptf_indices'][vlan_port])
+        lower_tor_standby_port = lower_tor_mg_facts['minigraph_vlans'][vlan_interface]['members'][0]
+        lower_tor_standby_server_ptf_port = lower_tor_mg_facts['minigraph_ptf_indices'][lower_tor_standby_port]
+
+    with allure.step("Switch the last server on the lower tor to active"):
+        toggle_simulator_port_to_lower_tor(lower_tor_active_port)
+        wait_until(10, 5, 0, check_muxcable_status, upper_tor_host, lower_tor_active_port, "standby")
+        wait_until(10, 5, 0, check_muxcable_status, lower_tor_host, lower_tor_active_port, "active")
+
+    with allure.step("Clear port counters"):
+        upper_tor_host.command("sonic-clear counters")
+        lower_tor_host.command("sonic-clear counters")
+
+    with allure.step("Start the traffic test, "
+                     "check the traffic received by the last server is the bounced back traffic from the upper tor"):
+        pkt = simple_tcp_packet(
+            eth_dst=dualtor_meta['vlan_mac'],
+            ip_src='1.1.1.1',
+            ip_dst=lower_tor_active_server_ip,
+            ip_ttl=64
+        )
+        # Send packets
+        ptfadapter.dataplane.flush()
+        testutils.send(ptfadapter, lower_tor_standby_server_ptf_port, pkt, PKT_NUM)
+        # The ttl of the bounced back packet should be decreased by 2
+        pkt.ttl -= 2
+        expected_packet = get_masked_packet(pkt)
+        # Check the bounced back packet is received by the last server ptf port
+        testutils.verify_packet(ptfadapter, expected_packet, lower_tor_active_server_ptf_port)
+        pkt.ttl += 2
+
+        # Send the packets again
+        ptfadapter.dataplane.flush()
+        testutils.send(ptfadapter, lower_tor_standby_server_ptf_port, pkt, PKT_NUM)
+        # The ttl of the packet directly forwarded by the lower tor should be decreased by 1
+        pkt.ttl -= 1
+        not_expected_packet = get_masked_packet(pkt)
+        # Check there is no such packet received by any ptf ports
+        testutils.verify_no_packet_any(
+            ptfadapter, not_expected_packet, lower_tor_t1_ptf_ports + lower_tor_vlan_ptf_ports)
+
+    with allure.step("Check the port counters to make sure the drop on lower tor happened"):
+        # There are in total 2 * PKT_NUM packets sent, the tx counter value of the last server port on the lower tor
+        # should be slightly lager than 2 * PKT_NUM, which means no duplicated packets are received by the server.
+        counter_range = [PKT_NUM * 2, PKT_NUM * 2 * 1.05]
+
+        def _check_counters():
+            upper_tor_port_counters = json.loads(upper_tor_host.get_port_counters(in_json=True))
+            lower_tor_port_counters = json.loads(lower_tor_host.get_port_counters(in_json=True))
+            upper_tor_active_port_rx_counter = int(
+                upper_tor_port_counters[lower_tor_standby_port]['RX_OK'].replace(',', ''))
```

Comment:
I suggest checking the standby port on upper_tor as well. The tx counter should be 0 (ignore noise).

Reply:
Will do.

Reply:
Done.